### DISCUSSION TITLE
Does WikiJs permit searches on assets contents?

### DISCUSSION THREAD

**Discussion by antonio-fasolato** [requarks/wiki#6394] [Help / Questions] [answered]
Is there a way to index the content of the assets (pdf, docx etc) together with the pages contents? I could use postgres, ElasticSearch or any search engine.

The documentation says nothing about it and the few test I did seems to confirm thata the assets are not indexed.

Thanks

**Comment by NGPixel** [comment, marked as answer]
No, only pages can be searched at the moment.
